### ISSUE TITLE
IvrAction not processing  number 0 

### ISSUE THREAD

**Issue by gil-obradors** [irontec/ivozprovider#1041]
Is posible that number 0 is not being correctly interpreted?
Other numbers are OK , only happens with 0

```
       > [b1][8a91d23c] [IvrAction.php:62         ] AGI Notice("PJSIP/proxytrunks-00003251", "Processing IVR IVR_in [Ivr#3]")
    -- AGI Script Executing Application: (Read) Options: (PRESSED,/opt/irontec/ivozprovider/storage/ivozprovider_model_locutions.encodedfile/0/8,1,,,6)
    -- Accepting a maximum of 1 digits.                                                                          
       > 0x7f9a100abdf0 -- Probation passed - setting RTP source address to *********************                              
    -- <PJSIP/proxytrunks-00003251> Playing '/opt/irontec/ivozprovider/storage/ivozprovider_model_locutions.encodedfile/0/8.slin' (language 'es')      
    -- Channel PJSIP/b1c7r2_pepitopalotes-0000324f left 'native_rtp' basic-bridge <01dcd0a8-9954-4121-a099-409457bb630d>
    -- Channel PJSIP/proxytrunks-0000324e left 'native_rtp' basic-bridge <01dcd0a8-9954-4121-a099-409457bb630d>
  == Spawn extension (call-residential, +34938750642, 2) exited non-zero on 'PJSIP/proxytrunks-0000324e'                                         
[Jun 28 16:49:02] DTMF[11822][C-000006b1]: channel.c:4126 __ast_read: DTMF begin '0' received on PJSIP/proxytrunks-00003251
[Jun 28 16:49:02] DTMF[11822][C-000006b1]: channel.c:4130 __ast_read: DTMF begin ignored '0' on PJSIP/proxytrunks-00003251
[Jun 28 16:49:02] DTMF[11822][C-000006b1]: channel.c:4040 __ast_read: DTMF end '0' received on PJSIP/proxytrunks-00003251, duration 260 ms
[Jun 28 16:49:02] DTMF[11822][C-000006b1]: channel.c:4110 __ast_read: DTMF end passthrough '0' on PJSIP/proxytrunks-00003251
    -- User entered '0'
       > [b1][8a91d23c] [IvrAction.php:72         ] AGI Verbos("PJSIP/proxytrunks-00003251", "IVR: User entered: 0")
       > [b1][8a91d23c] [IvrAction.php:164        ] AGI Verbos("PJSIP/proxytrunks-00003251", "Processing IVR no input handler.")
       > [b1][8a91d23c] [ExtensionAction.php:55   ] AGI Notice("PJSIP/proxytrunks-00003251", "Processing Extension 999 [Extension#55]")
```

other example
```
 > [b1][1ff57b98] [IvrAction.php:62         ] AGI Notice("PJSIP/proxytrunks-00000411", "Processing IVR IVR_in [Ivr#3]")
    -- AGI Script Executing Application: (Read) Options: (PRESSED,/opt/irontec/ivozprovider/storage/ivozprovider_model_locutions.encodedfile/0/8,4,,,6)
    -- Accepting a maximum of 4 digits.                                                                                                
       > 0x7f9a08069600 -- Probation passed - setting RTP source address to *********                          
    -- <PJSIP/proxytrunks-00000411> Playing '/opt/irontec/ivozprovider/storage/ivozprovider_model_locutions.encodedfile/0/8.slin' (language 'es')
[Jun 19 09:16:24] DTMF[9173][C-000001ed]: channel.c:4126 __ast_read: DTMF begin '0' received on PJSIP/proxytrunks-00000411                     
[Jun 19 09:16:24] DTMF[9173][C-000001ed]: channel.c:4130 __ast_read: DTMF begin ignored '0' on PJSIP/proxytrunks-00000411
[Jun 19 09:16:24] DTMF[9173][C-000001ed]: channel.c:4040 __ast_read: DTMF end '0' received on PJSIP/proxytrunks-00000411, duration 260 ms
[Jun 19 09:16:24] DTMF[9173][C-000001ed]: channel.c:4110 __ast_read: DTMF end passthrough '0' on PJSIP/proxytrunks-00000411            
    -- User entered '0'                                                                                                                         
       > [b1][1ff57b98] [IvrAction.php:72         ] AGI Verbos("PJSIP/proxytrunks-00000411", "IVR: User entered: 0")
       > [b1][1ff57b98] [IvrAction.php:164        ] AGI Verbos("PJSIP/proxytrunks-00000411", "Processing IVR no input handler.")         
```

 2.10~2.10.0~20190430.2009~1db7b82+artemis all

**Comment by Kaian** [comment]
This seems a problem with the function used to check the entered value. For that function 0 is actually an empty value.

https://github.com/irontec/ivozprovider/blob/bleeding/asterisk/agi/src/Agi/Action/IvrAction.php#L79-L83

https://php.net/empty

Thanks for reporting!

**Comment by Kaian** [comment]
Fixed in 8806f153f for 2.11.1
